### DISCUSSION TITLE
Server-Side Auth cookies expose sensitive information

### DISCUSSION THREAD

**Discussion by EduardKakosyan** [supabase/supabase#30491] [Questions]
Hey everyone! I’m fairly new to Supabase, so I apologize if this is a basic question.

I’ve followed the documentation to set up Server-Side Auth persistence, and it’s working well. The access and refresh tokens are stored correctly. However, it seems to also store all the user information from my Supabase table, mainly it shows the UUID associated with the user, which I don't want to have in the cookie. Is there a way to prevent this while using the Supabase auth method? If so, could you please share a guide or tutorial that explains how to do this?

[Guide](https://supabase.com/docs/guides/auth/server-side/nextjs?queryGroups=router&router=app).

I’ve also used the Vercel + Supabase [template](https://vercel.com/templates/next.js/supabase), and the same thing happens, so I’m confident it’s not an issue with my code. I even tested it on a completely fresh Supabase project, and the problem persists.

Any help is appreciated.
 Thanks!

**Comment by GaryAustin1** [comment]
The cookie normally contains the supabase session which includes all the user info including that from the JWT (which also can be decoded to get the similar info).  The UUID is not considered to be secret info.
